### PR TITLE
Add filter by resource type to version get request

### PR DESCRIPTION
**Pull request by Henya14**

### CODE REVIEW

File: core/com.b2international.snowowl.core.rest/src/com/b2international/snowowl/core/rest/codesystem/VersionRestSearch.java [hunk 1]
```diff
@@ -29,12 +30,23 @@
 	@Parameter(description="The corresponding resource identifier(s) to match")
 	private List<String> resource;
 	
+	@Parameter(description = "The types of resources to get the versions for", in = ParameterIn.QUERY)
```
Comment (cmark) [inline core/com.b2international.snowowl.core.rest/src/com/b2international/snowowl/core/rest/codesystem/VersionRestSearch.java]: The `in` value is not required, please remove it. All parameters are being registered as query parameters by default.

**Review by cmark — CHANGES_REQUESTED** [review]

File: core/com.b2international.snowowl.core.rest/src/com/b2international/snowowl/core/rest/codesystem/VersionRestSearch.java (hunk 1 above)
Comment (Henya14) [reply core/com.b2international.snowowl.core.rest/src/com/b2international/snowowl/core/rest/codesystem/VersionRestSearch.java]: I'm following the open API spec, which states the `in` parameter is required. The default value for `in` is `ParameterIn.Default`.
https://spec.openapis.org/oas/v3.1.0#parameter-object
image

Comment (cmark) [reply core/com.b2international.snowowl.core.rest/src/com/b2international/snowowl/core/rest/codesystem/VersionRestSearch.java]: Yes, that is required in the generated OAS3 spec JSON output, but the value is being added automatically by the `springdoc` generator, so you don't have to add it explicitly. Please remove it to be consistent with the other annotations. 🙇

Comment (Henya14) [reply core/com.b2international.snowowl.core.rest/src/com/b2international/snowowl/core/rest/codesystem/VersionRestSearch.java]: Done

**Review by kovacstomi — APPROVED** [review]
LGTM!

File: snomed/com.b2international.snowowl.snomed.core.rest.tests/src/com/b2international/snowowl/snomed/core/rest/components/SnomedRefSetMemberApiTest.java [hunk 2]
```diff
@@ -800,8 +800,7 @@ public void executeInvalidAction() throws Exception {
 			"commitComment", "Executed invalid action on reference set member"
 		);
 
-		executeMemberAction(branchPath, memberId, invalidActionRequest).statusCode(400)
-		.body("message", CoreMatchers.equalTo("Invalid action type 'invalid'."));
+		executeMemberAction(branchPath, memberId, invalidActionRequest).statusCode(400);
```
Comment (cmark) [inline snomed/com.b2international.snowowl.snomed.core.rest.tests/src/com/b2international/snowowl/snomed/core/rest/components/SnomedRefSetMemberApiTest.java]: If there is a new error message in the response then please assert that.

**Review by cmark — CHANGES_REQUESTED** [review]

File: snomed/com.b2international.snowowl.snomed.core.rest.tests/src/com/b2international/snowowl/snomed/core/rest/components/SnomedRefSetMemberApiTest.java (hunk 2 above)
Comment (Henya14) [reply snomed/com.b2international.snowowl.snomed.core.rest.tests/src/com/b2international/snowowl/snomed/core/rest/components/SnomedRefSetMemberApiTest.java]: Done

**Review by cmark — APPROVED** [review]
LGTM!

**Pull request merged**